### PR TITLE
Feat: option for out of the box minikube and kind via gen-config

### PR DESCRIPTION
**Pull request by Avi-Robusta**
we wanted to change the size of memory consumption of all our pods to work on smaller clusters for testing

### CODE REVIEW

**Comment by Avi-Robusta** [comment]
- [ ] need to test final time on minikube
- [ ] need to test final time  on kind

**Comment by Avi-Robusta** [comment]
[] for the test robusta gen-config you need to add the flag --no-is-small-cluster

File: src/robusta/cli/main.py [hunk 1]
```diff
@@ -29,6 +29,10 @@
     replace_in_file,
 )
 
+FORWARDER_CONFIG_FOR_SMALL_CLUSTERS = "256Mi"
```
Comment (arikalon1) [inline src/robusta/cli/main.py:32]: I think 64MB should be enough here

File: src/robusta/cli/main.py [hunk 2]
```diff
@@ -29,6 +29,10 @@
...
+RUNNER_CONFIG_FOR_SMALL_CLUSTERS = "512Mi"
+GRAFANA_CONFIG_FOR_SMALL_CLUSTERS = "256Mi"
```
Comment (arikalon1) [inline src/robusta/cli/main.py:34]: I think 64MB should be enough here as well. (I just looked at the memory chart for both on small clusters, and it was less than 64MB)
In addition, I would name it GRAFANA_RENDERER , cause grafana is a different pod. might be confusing

File: src/robusta/cli/main.py [hunk 3]
```diff
@@ -47,14 +51,29 @@ class GlobalConfig(BaseModel):
     account_id: str = ""
 
 
+class PodConfigs(Dict[str, Dict[str, Dict[str, str]]]):
+    __root__: Dict[str, Dict[str, Dict[str, str]]]
+
+    @classmethod
+    def gen_config(cls, memory_size: str = "256Mi"):
```
Comment (arikalon1) [inline src/robusta/cli/main.py:58]: Why do we need a default here?

File: src/robusta/cli/main.py [hunk 4]
```diff
@@ -99,6 +119,10 @@ def gen_config(
             "Please specify a unique name for your cluster or press ENTER to use the default",
             default=guess_cluster_name(),
         )
+    if is_small_cluster is None:
+        is_small_cluster = typer.confirm(
+            "Are you running a small local cluster? (Like minikube or kind)"
```
Comment (arikalon1) [inline src/robusta/cli/main.py:124]: Will "Are you running a mini cluster? (Like ...)" sound better?

**Comment by arikalon1** [comment]
Good job.
Few minor comments

File: src/robusta/cli/main.py (hunk 2 above)
Comment (Avi-Robusta) [reply src/robusta/cli/main.py:34]: when you mean for both do you mean robusta runner too?

Comment (arikalon1) [reply src/robusta/cli/main.py]: No, The runner does need 512MB
The renderer and forwarder will be able to work with 64MB

**Pull request merged**